### DISCUSSION TITLE
Solving NEMOH using BEM Rosetta

### DISCUSSION THREAD

**Discussion by blackhydra** [BEMRosetta/BEMRosetta#45] [Q&A] [answered]
Hello,

Thanks for developing and updating BEMRosetta,

I am a member of research team to preliminary analyse the NEMOH solver, and I have questions regarding the use of BEMRosetta as a solver for NEMOH. It shows that the BEMRosetta has "BEM Solver" tab, I have been trying to use it with an example from BEMRosetta resources (Github).

1. What is the output for this BEMRosetta solver?
2. Does it produce the same output as NEMOH as an application? Added Mass, Radiation Damping, RAO, etc.

Thank you for your time and effort.

Regars,
Ahmad

![error bin file](https://github.com/BEMRosetta/BEMRosetta/assets/17959317/1dda48ec-b03f-4eef-981b-a32b2213f665)

**Comment by izabala123** [comment, marked as answer]
Hi Ahmad

This tab is not for running any solvers, but for preparing the files to be run by the supported solvers.
All the files are saved in a folder.
As for Nemoh and HAMS the binaries are included in the folder, you have to set the location of them in the Options tab.
For example, this is my Options tab:

![image](https://github.com/BEMRosetta/BEMRosetta/assets/38589221/6a00b59c-652b-442e-9374-92791d21085f)

Best regards
Iñaki

**Reply by blackhydra** [reply]
Hi izabala123,

Thank you for clearing the confusion, helps me a lot, It was quite challenging to understand BEM Rosetta without the documentation. I know it is not directly related to NEMOH, If you have a documentation about this, please do kindly refer me the link or files.

For anyone whom cannot produce the preprocessing files using NEMOH, I found that i have to uncheck the "include the solver binaries". Hope it helps.

I really appreciate the time and effort.

Regards,
Ahmad
